framework/components/ecl-profile-topbars/ecl-profile-topbars.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * ECL Profile Topbar
+ * @define profile-topbar
+ */
+
+.ecl-profile-topbar {
+  margin: 0;
+}
+
+// This is esentially .section__group which can be used when refactored.
+.ecl-profile-topbar__wrapper {
```

Comment:
I'm not sure I understand what you say..
My remark was: instead of using external margins on a wrapper, we should probably use padding on the main .ecl-profile-topbar element.
That way, there won't be anything going outside the component.
And also if we do so, the wrapper itself becomes useless (no more css attributes on it)